.github/workflows/check-typos.yml

```diff
@@ -0,0 +1,21 @@
+name: Spell Check
+on: [pull_request]
+
+jobs:
+  run:
+    name: typos
+    runs-on: "ubuntu-20.04"
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+
+    - name: Check spelling
+      uses: crate-ci/typos@v1.2.1
+
+    - name: Help
+      if: always()
+      run: |
+        echo 'To run the check locally, install typos from https://github.com/crate-ci/typos and run `typos` in the repository.'
+        echo 'Run `typos -w` to automatically correct spelling mistakes.'
```

Comment:
Don't we have to escape those ticks like in https://github.com/GPUOpen-Drivers/llpc/pull/1495/files?

Reply:
No, it’s single quotes here, so it works: https://github.com/GPUOpen-Drivers/llpc/runs/4191968589?check_suite_focus=true